samples/javascript_nodejs/02.echobot-with-counter/bot.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityTypes } = require('botbuilder');
+
+// Turn counter property
+const TURN_COUNTER_PROPERTY = 'turnCounterProperty';
+
+class Bot {
```

Comment:
"EchoBot"? I've been naming my bots "AdaptiveCardsBot" and "RichCardsBot"

Reply:
Leaving this as Bot. This way we can be consistent across the samples.

Reply:
Take that back. I like EchoBot better. Fixed.